### PR TITLE
CDAP-2422: Remove redundant catch blocks in AdapterHttpHandler

### PR DESCRIPTION
**Pull request by jwang47**
https://issues.cask.co/browse/CDAP-2422
http://builds.cask.co/browse/CDAP-DUT1812

Preview before making changes to the rest of the HTTP handlers.

### CODE REVIEW

**Comment by rjain** [comment]
Hi Alvin,

Did you intend to assign this one to me ? If yes, I probably will need more details on what you expect from me :).

**Comment by jwang47** [comment]
Sorry, that was probably a mistake, wasn't intending to assign to anyone.

File: cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java [hunk 1]
```diff
@@ -327,41 +332,68 @@ public void createAdapter(Id.Adapter id, AdapterConfig config) {
...
-    adapterHttpHandler.getAdapterRuns(request, responder, id.getNamespaceId(), id.getId(), null, null, null, 100);
-    verifyResponse(HttpResponseStatus.valueOf(200), responder.getStatus(), "Failed to get runs for adapter.");
-    return responder.decodeResponseContent(RUN_RECORDS_TYPE);
+
+    try {
+      adapterHttpHandler.getAdapterRuns(request, responder, id.getNamespaceId(), id.getId(), null, null, null, 100);
+      verifyResponse(HttpResponseStatus.valueOf(200), responder.getStatus(), "Failed to get runs for adapter.");
+      return responder.decodeResponseContent(RUN_RECORDS_TYPE);
+    } catch (Exception e) {
+      LOG.error("Error getting adapter runs", e);
```
Comment (henry-cask) [inline cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java]: Extra space between adapter and runs?

File: cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java [hunk 2]
```diff
@@ -327,41 +332,68 @@ public void createAdapter(Id.Adapter id, AdapterConfig config) {
...
-    adapterHttpHandler.getAdapterRun(request, responder, adapterId.getNamespaceId(), adapterId.getId(), runId);
-    verifyResponse(HttpResponseStatus.valueOf(200), responder.getStatus(), "Failed to get run for adapter.");
-    return responder.decodeResponseContent(RunRecord.class);
+
+    try {
+      adapterHttpHandler.getAdapterRun(request, responder, adapterId.getNamespaceId(), adapterId.getId(), runId);
+      verifyResponse(HttpResponseStatus.valueOf(200), responder.getStatus(), "Failed to get run for adapter.");
+      return responder.decodeResponseContent(RunRecord.class);
+    } catch (Exception e) {
+      LOG.error("Error getting adapter run", e);
```
Comment (henry-cask) [inline cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java]: Extra space between adapter and runs?

**Comment by henry-cask** [comment]
Did a pass and other than small nits on log messages +1 LGTM

File: cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java (hunk 1 above)
Comment (jwang47) [inline cdap-app-fabric/src/main/java/co/cask/cdap/test/internal/AppFabricClient.java]: Don't see an extra space. Can you clarify?

**Pull request merged**